styles.css

```diff
@@ -0,0 +1,112 @@
+body {
+  height: 100%;
+  max-width: 100%;
+  font-family: "Roboto", sans-serif;
+}
+nav {
+  top: 0px;
+  width: 100%;
+  display: flex;
+}
+#youtube-logo {
+  width: auto;
+  height: 24px;
+  align-self: flex-start;
+}
+.search {
+  margin-left: auto;
+  margin-right: auto;
+}
+#search-input {
+  width: 400px;
+  height: 24px;
+}
+#search-button {
+  width: auto;
+  height: 18px;
+}
+#button {
+  width: auto;
+  height: 24px;
+}
+.nav-items {
+  width: auto;
+  height: 24px;
+  margin-left: 1.2%;
+}
+.video-container {
+  float: left;
+  width: 70%;
+  margin-left: 1.2%;
+}
+.video {
+  margin-top: 3%;
+}
+#iframe {
+  width: 100%;
+  height: 450px;
+}
+.subvideo {
+  width: 100%;
+}
+#title {
+  font-size: 20px;
+}
+.leftside {
+  display: inline-block;
+}
+#view-count {
+  font-size: 15px;
+  color: rgb(107, 107, 107);
+}
+#avatar {
+  width: 24px;
+  height: 24px;
+}
+.rightside {
+  float: right;
+}
+#like {
+  width: 24px;
+  height: 24px;
+}
+#dislike {
+  width: 24px;
+  height: 24px;
+  margin-left: 10px;
+}
```

Comment:
You are repeating width: 24px; height: 24px; two times, use a class selector instead for that